beacon/src/entropy.rs

```diff
@@ -36,6 +36,18 @@ impl Entropy {
         })
     }
 
+    pub fn from_data(data: Vec<u8>) -> Result<Self> {
```

Comment:
is there any way for this to be (or _should_ this be) size bound in the way that the `local()` entropy Vec is bound by the size of the `LOCAL_ENTROPY_SIZE` const?

Reply:
Whether or not it makes sense to be bounded, it does make sense to be a `Vec` since this fn is taking ownership.